components/eamxx/docs/user/model_input.md

```diff
@@ -1,8 +1,269 @@
+The main files/scripts involved in the EAMxx runtime configuration are the following:
 
+1. `buildnml`: this script, located in `components/eamxx/cime_config`, is called by CIME's case management
+    scripts (`case.setup`, `case.build, `case.submit`), and is responsible for creating EAMxx input files,
+    including handling of user-specific modifications for the runtime parameters. Users should not have to
+    modify this script, nor should they have to manually call it.
+2. `scream_input.yaml`: EAMxx will parse this [YAML](https://yaml.org/spec/1.2.2) file at runtime,
+    to detect all of its configuration parameters. It is generated automatically by `buildnml` inside the
+    RUNDIR/data folder, where `RUNDIR` is the run directory generated by CIME. Since this file is automatically
+    generated when `buildnml` runs, users should not manually modify it. Any manual modification will be
+    lost the next time `buildnml` runs (e.g., at `case.submit` time).
+3. 'namelist_scream.xml`: this XML file is generated by `buildnml`, and contains all the runtime parameters that EAMxx
+    will read in at runtime. `buildnml` uses this XML file as an intermediate file during the generation of
+    `scream_input.yaml`. In particular, `buildnml` generates this file using case information to select the
+    proper configuration from the file `namelist_defaults_scream.xml`, located in `components/eamxx/cime_config`.
+    As it was the case for `scream_input.yaml`, since this XML file is automatically generated by `buildnml`,
+    the user should not manually modify this file. Any manual modification will be lost the next time `buildnml`
+    runs (e.g., at `case.submit` time).
+    Since the main product of `buildnml` is `scream_input.yaml`, one may wonder why we keep
+    this XML file around at all. The main reason is to make it easier for the `atmquery` script (see below) to
+    retrieve settings.
+4. `namelist.nl`: EAMxx will parse this [namelist](https://docs.oracle.com/cd/E19957-01/805-4939/6j4m0vnad/index.html)
+    file at runtime, to retrieve all the parameters specific to the Homme (ADD REF) dycore.
+    It is generated automatically by `buildnml` inside the RUNDIR/data folder, where `RUNDIR` is the run
+    directory generated by CIME. Since this file is automatically generated when `buildnml` runs,
+    users should not manually modify it. Any manual modification will be lost the next time `buildnml` runs
+    (e.g., at `case.submit` time).
+5. `atmchange` and `atmquery`: these script, located in `components/eamxx/scripts` and linked in the case folder,
+    can be used to query and change any of the runtime configuration parameters of EAMxx.
 
+Among the files listed above, `atmquery` and `atmchange` are the only ones that the user should interact with,
+and therefore we give here a brief overview of how they can be used, and how their output can be interpreted.
```

Comment:
Maybe list atmchange and atmquery before the files nobody should touch so users see what they need before reading stuff they don't really need to know?

Also, I feel like the scream_input.yaml, namelist_scream.xml, and namelist.nl descriptions could be explained in an easier to digest way by saying something like "buildnml also generates 3 files which the model uses to choose options. These files are... Users should not directly modify any of these files... .yaml is used by the model, .xml is needed to interface with atmquery and atmchange, and namelist.nl contains dycore-specific options."